genesyscloud/journey_outcome_predictor/resource_genesyscloud_journey_outcome_predictor_test.go

```diff
@@ -0,0 +1,91 @@
+package journey_outcome_predictor
+
+import (
+	"fmt"
+	"terraform-provider-genesyscloud/genesyscloud/provider"
+	"terraform-provider-genesyscloud/genesyscloud/util"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/mypurecloud/platform-client-sdk-go/v123/platformclientv2"
+)
+
+func TestAccResourceJourneyOutcomePredictor(t *testing.T) {
+	t.Parallel()
+	var (
+		fullResourceName = "genesyscloud_journey_outcome_predictor.test_predictor"
+		fullOutcomeResourceName = "genesyscloud_journey_outcome.test_outcome"
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { util.TestAccPreCheck(t) },
+		ProviderFactories: provider.GetProviderFactories(providerResources, providerDataSources),
+		Steps: []resource.TestStep{
+			{
+				Config: predictorResource(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrPair(fullResourceName, "outcome_id", fullOutcomeResourceName, "id"),
+				),
+			},
+			{
+				ResourceName:            fullResourceName,
+				ImportState:             true,
+				ImportStateVerify:       false,
+			},
+		},
+		CheckDestroy: testVerifyPredictorDestroyed,
+	})
+
+}
+
+func predictorResource() string {
```

Comment:
I got an error when I tried to run this test: `Invalid multi-line string`, I'm not really sure why this is happening but it's most likely due to how the resources are being made i.e. all in one string. Instead it will probably be easier to have two functions, one to make the the journey_outcome and then one to make the journey_outcome_predictor.resource_genesyscloud_architect_grammar_language_test.go might be a good file to look at for an example. It is also probably best if you name them `generateXResource` to be consistent with the other test functions used to generate resources